assets/src/edit-story/components/form/colorInput.js

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { toState } from 'react-color/lib/helpers';
+
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import ColorPicker from '../colorPicker';
+import Group from './group';
+
+const Preview = styled.button`
+	width: 18px;
+	height: 18px;
+	padding: 0;
+	background: ${ ( { value } ) => value };
+	border: 1px solid #d3d4d4;
+`;
+
+const Picker = styled( ColorPicker )`
+	position: absolute;
+	top: 18px;
+`;
+
+const Wrap = styled.div`
+	position: relative;
+`;
+
+const VisuallyHidden = styled.span`
+	border: 0;
+	clip: rect(1px, 1px, 1px, 1px);
+	clip-path: inset(50%);
+	height: 1px;
+	margin: -1px;
+	overflow: hidden;
+	padding: 0;
+	position: absolute;
+	width: 1px;
+	word-wrap: normal !important;
+
+	&:focus {
+		background-color: $light-gray-500;
```

Comment:
Where does `$light-gray-500` come from?

Reply:
Whoops 😄 from WordPress. Need to remove that.